pkg/operator/controller/ingress/deployment_test.go

```diff
@@ -901,6 +930,7 @@ func TestDeploymentConfigChanged(t *testing.T) {
 						},
 					},
 					Spec: corev1.PodSpec{
+						DNSPolicy: corev1.DNSClusterFirstWithHostNet,
```

Comment:
Would it make more sense to set the test deployment's `DNSPolicy` to `corev1.DNSClusterFirst` since that's the API defaulted value (and then mutate that value to `corev1.DNSClusterFirstWithHostNet` in the new test cases)?